### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by fsocie7y**
Screenshot_2
Screenshot_1
Screenshot_3
Screenshot_4
Screenshot_5

### CODE REVIEW

**Review by Mowka7 — CHANGES_REQUESTED** [review]
Everything is okay, but add new lines everythere

**Comment by Mowka7** [comment]
> > 
> 
> New line everywhere present, it's just GitHub, cuted they)

got it, sorry

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.idea/
+.vscode/
+*.iml
+.env
+.DS_Store
+venv/
+.pytest_cache/
+**__pycache__/
+*.pyc
+**db.sqlite3
```
Comment (TPolina) [inline .gitignore:10]: add blank line to the end of the files

**Review by TPolina — CHANGES_REQUESTED** [review]
delete pycache from GitHub

**Review by TPolina — CHANGES_REQUESTED** [review]
delete .pyc files from GitHub

**Pull request closed without merging**